Build/Build.cs

```diff
@@ -138,32 +136,35 @@ class Build : NukeBuild
                     cc => cc.SetProjectFile(Solution.Specs.Approval_Tests)));
         });
 
-    Target UnitTests => _ => _
+    Project[] Projects => new[]
+    {
+        Solution.Specs.FluentAssertions_Specs,
+        Solution.Specs.FluentAssertions_Equivalency_Specs
+    };
+
+    Target UnitTestsNetFramework => _ => _
+        .Unlisted()
         .DependsOn(Compile)
-        .OnlyWhenDynamic(() => RunAllTargets || HasSourceChanges)
+        .OnlyWhenDynamic(() => EnvironmentInfo.IsWin && (RunAllTargets || HasSourceChanges))
         .Executes(() =>
         {
-            Project[] projects = new[]
-            {
-                Solution.Specs.FluentAssertions_Specs,
-                Solution.Specs.FluentAssertions_Equivalency_Specs
-            };
-
-            if (EnvironmentInfo.IsWin)
-            {
-                Xunit2(s =>
-                {
-                    IEnumerable<string> testAssemblies = projects
-                        .SelectMany(project => GlobFiles(project.Directory, "bin/Debug/net47/*.Specs.dll"));
-
-                    Assert.NotEmpty(testAssemblies.ToList());
-
-                    return s
-                        .SetFramework("net47")
-                        .AddTargetAssemblies(testAssemblies);
-                });
-            }
+            IEnumerable<string> testAssemblies = Projects
+                    .SelectMany(project => GlobFiles(project.Directory, "bin/Debug/net47/*.Specs.dll"));
+            
+            Assert.NotEmpty(testAssemblies.ToList());
+
+            Xunit2(s => s
+                .SetFramework("net47")
+                .AddTargetAssemblies(testAssemblies)
+            );
+        });
 
+    Target UnitTestsNetCore => _ => _
+        .Unlisted()
+        .DependsOn(Compile)
+        .OnlyWhenDynamic(() => RunAllTargets || HasSourceChanges)
+        .Executes(() =>
+        {
             DotNetTest(s => s
                 .SetConfiguration("Debug")
```

Comment:
I also would like to use `Configuration.Debug` (the enumeration class) to get type safety

Also applies to other appearances